README.md

```diff
@@ -55,7 +55,7 @@ Currently this is handled by running a script in the [onboarding/ dir of the vir
 Important: The database docker-compose must be up and running with all migrations applied, as described above, 
 BEFORE running any of the following steps.
 
-In addition, create a `.env` file in the root of this repo with an environment variable called `DB_HOST`
+In addition, create a `.env` file in the root of this repo with an environment variable called `DATABASE_URL`
 that refers to the endpoint of the database (usually `localhost:5432` if you're running things locally). 
```

Comment:
Probably this information now needs to be updated right? Because you now need to pass the full url? Please update documentation with the sensible defaults for developers.

Reply:
I'm here referring to 'usually `localhost:5432` if you're running things locally'.

Reply:
Agreed, I've updated the URL and made clear in the text that it assumes you're running it with (one of) the docker-compose yamls.